src/quo2/components/wallet/account_overview/component_spec.cljs

```diff
@@ -0,0 +1,36 @@
+(ns quo2.components.wallet.account-overview.component-spec
+  (:require [quo2.components.wallet.account-overview.view :as account-overview]
+            [test-helpers.component :as h]))
+
+(h/describe "Account overview test"
+  (h/test "renders correct account name"
+    (h/render [account-overview/account-overview
+               {:account-name "Diamond Hand"}])
+    (-> (js/expect (h/get-by-text "Diamond Hand"))
+        (.toBeTruthy)))
+
+  (h/test "renders correct account value"
+    (h/render [account-overview/account-overview
+               {:current-value "20.0$"}])
+    (-> (js/expect (h/get-by-text "20.0$"))
```

Comment:
@ulisesmac added a nice helper, h/is-truthy. There are some examples you can follow to use it :)

Reply:
Done